tests/main/refresh/task.yaml

```diff
@@ -129,7 +123,7 @@ execute: |
     fi
 
     echo "When the snap is refreshed"
-    snap refresh --ignore-running --channel=edge "$SNAP_NAME"
```

Comment:
Was this flag actually necessary before? Or was it just a "to be safe" thing?

Reply:
IIRC just to be on the safe side but I could misremember